Terning-iOS/Terning-iOS/Source/Presentation/JobDetail/Cell/CompanyInfoTableViewCell.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  CompanyInfoTableViewCell.swift
+//  Terning-iOS
+//
+//  Created by 정민지 on 7/12/24.
+//
+
+import UIKit
+
+import SnapKit
+
+final class CompanyInfoTableViewCell: UITableViewCell {
+    
+    // MARK: - UI Components
+    
+    private let companyImageView = UIImageView().then {
+        $0.contentMode = .scaleAspectFill
+        $0.layer.cornerRadius = 30
+        $0.layer.masksToBounds = true 
+        $0.layer.borderColor = UIColor.terningMain.cgColor
+        $0.layer.borderWidth = 1
```

Comment:
`makeBorder` 활용 바랍니다. !

```suggestion
        $0.layer.masksToBounds = true 
        $0.makeBorder(width: 1, color: .terningMain)
```

Reply:
여기도 확인했습니다~!